app/javascript/components/batch_update/helpers/elasticsearch.js

```diff
@@ -107,6 +116,15 @@ const buildFilterMatches = ({ publishedFilter, genomedFilter }) => [
   genomedMatcher(genomedFilter),
 ]
 
+const buildPriceMatch = ({ minPrice, maxPrice }) => ({
+  range: {
+    prices: {
+      gte: minPrice,
+      lte: maxPrice,
```

Comment:
One or the other of these may be null in this code path, correct? But this just works anyway?

Reply:
It works correctly as long as any unset values are explicitly null, yeah! :)